### PR TITLE
Electron 48 : launch app fully sandboxed

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
@lneir Please review and merge.

CC: @VikasShashidhar

### CODE REVIEW

**Comment by VishwasShashidhar** [comment]
FYI, Symphony.exe is the executable (created from the batch file) that opens our actual electron executable.

**Comment by VishwasShashidhar** [comment]
Changes w.r.t. auto-updates will be pushed in the electron-16 branch

**Comment by VishwasShashidhar** [comment]
This PR includes changes for both mac and windows.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Symphony",
-  "productName": "Symphony",
+  "productName": "Symphony-Electron",
```
Comment (VikasShashidhar) [inline package.json]: I think this depends on Symphony's branding strategy for the Electron App and on whether the existing Desktop Wrapper will be continued to be used after the Electron wrapper has been released. We can probably keep this as "Symphony" only.

Same in other places as well.

@lneir Please share your inputs.

**Comment by VishwasShashidhar** [comment]
@lneir @VikasShashidhar 

I've fixed the product name issue raised above now. Instead, we would now rename the generated exe as Symphony-Electron.exe.

Note that this will not be the user facing exe, it'll be the binary launched from the batch file (converted to exe).

Please review now.

**Comment by VikasShashidhar** [comment]
LGTM 👍

**Pull request merged**